sdk/translation/Azure.AI.Translation.Document/CHANGELOG.md

```diff
@@ -1,14 +1,16 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 2.0.0-beta.1 (2024-05-07)
 
 ### Features Added
+- Added Synchronous document translation API: {RESOURCE}/translator/document:translate?api-version=[Date]. 
```

Comment:
This line seems redundant now that you added the 2 below, I would remove it.

Reply:
Removed